### PR TITLE
Update spec editors

### PR DESCRIPTION
**Pull request by jyavenard**
- Add Jean-Yves Avenard as editor
- Move Matt Wolenetz to former editors

***
<a href="https://pr-preview.s3.amazonaws.com/jyavenard/media-source/pull/323.html" title="Last updated on Oct 10, 2023, 10:13 AM UTC (d60db94)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/media-source/323/898e0c1...jyavenard:d60db94.html" title="Last updated on Oct 10, 2023, 10:13 AM UTC (d60db94)">Diff</a>

### CODE REVIEW

**Review by dalecurtis — APPROVED** [review]
lgtm, thanks!

**Comment by chrisn** [comment]
As he's an IE in the group, I've pinged Matt to check he's OK with the change, so please don't merge just yet!

**Comment by jyavenard** [comment]
If Matt wants to continue being the editor it’s all fine by me. I only stepped in because someone had to.

File: media-source-respec.html [hunk 1]
```diff
@@ -30,15 +30,17 @@
       prevRecURI: "https://www.w3.org/TR/2016/REC-media-source-20161117/",
 
       editors:  [
-      { name: "Matthew Wolenetz",  mailto:"wolenetz@google.com", url: "",
```
Comment (wolenetz) [inline media-source-respec.html:33]: If there is an option for me to retain editorship (in addition to, of course, Jean-Yves being added as an editor), that would be my preference. In that case, @chrisn, how would organization be represented for me? (e.g., "W3C Media Working Group Invited Expert" with empty companyURL?) Tt would also be good to include me also in the formerEditors as shown in this change, to indicate the period of time where I was editing on behalf of Google.) What do you think? (And thank you for pinging me about this, too, @chrisn !).

**Comment by chrisn** [comment]
@jyavenard If you're OK with it, what I'd like to do is have you both, with Mark, as co-editors. We need your help too!

File: media-source-respec.html (hunk 1 above)
Comment (chrisn) [reply media-source-respec.html:33]: The W3C [style guide](https://w3c.github.io/manual-of-style/#affs) gives some examples, so I would suggest following those, and [this advice](https://github.com/w3c/respec/issues/2561#issuecomment-548184120), i.e:

```
{ name: "Matthew Wolenetz",  mailto: "", url: "", company: "Invited Expert",
  note: "and before at Google Inc.", w3cid: "76912" }
```

which renders as:

> Matthew Wolenetz (Invited Expert) (and before at Google Inc.)

You can include your contact email and URL, or omit these as you prefer. As you're still an editor, there's no need to move you to `formerEditors`.

Comment (dalecurtis) [reply media-source-respec.html:33]: We're happy to have Matt continue as an IE editor if he'd like!

**Comment by chrisn** [comment]
I've updated the PR to only add @jyavenard as editor. @wolenetz I suggest raising another PR to update your affiliation as discussed here.

**Pull request merged**